<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_logo{
background:white;
}

#frame{
background:rgba(255,255,255,200);
border-radius:25px;
border-style:solid;
border-width: 1px;
border-color:black;
}

#frame_similarity{
background:white;
border-radius:10px;
border-style:solid;
border-width: 1px;
border-color:black;
}

#frame_filter{
background:white;
border-radius:10px;
border-style:solid;
border-width: 1px;
border-color:black;
}

#frame_facedetection{
background:white;
border-radius:10px;
border-style:solid;
border-width: 1px;
border-color:black;
}


#label_line_1{
background:rgba(14,11,101,220);
color: white;
vertical-align: middle;
border-radius:13px;
}

#label_line_2{
background:rgba(14,11,101,220);
color: white;
vertical-align: middle;
border-radius:13px;
}

#label_title_before{
background:rgba(137,136,191,250);
color: white;
vertical-align: middle;
border-radius:5px;
padding-left: 10 px;
}


#label_title_after{
background:rgba(137,136,191,250);
color: white;
vertical-align: middle;
border-radius:5px;
padding-right: 10 px;
}

#label_title_similarity{
background:rgba(14,11,101,220);
color: white;
vertical-align: middle;
border-radius:3px;
}

#label_title_filter{
background:rgba(14,11,101,220);
color: white;
vertical-align: middle;
border-radius:3px;
}

#label_title_facedetection{
background:rgba(14,11,101,220);
color: white;
vertical-align: middle;
border-radius:3px;
}

QWidget{
background:url(:/GUI_Ressources/image_background3.png);
}

#label_choose_filetype{
background:rgba(27,27,27,30);
border-radius:3px;
color: black;
padding-left: 3px;
}


#label_choose_filter{
background:rgba(27,27,27,30);
border-radius:3px;
color: black;
padding-left: 3px;
}


#label_choose_facedetection{
background:rgba(27,27,27,30);
border-radius:3px;
color: black;
border-color:black;
border-width:1px;
padding-left: 3px;
}


#label_result{
background:rgba(27,27,27,30);
border-radius:3px;
color: black;
padding-left: 3px;
}


#label_result_display{
background:transparent;
border-style:solid;
border-width:1.2px;
border-color:rgba(27,27,27,100);
border-radius:3px;
color: black;
}


#label_aktualisieren{
background:rgba(137,136,191,250);
border-radius:3px;
color: white;
}

#label_aktualisieren:hover{
background:white;
border-radius:3px;
border-style:solid;
border-width:1.5px;
border-color:rgba(137,136,191,250);
color: black;
}

#label_filter_anwenden{
background:rgba(137,136,191,250);
border-radius:3px;
color: white;
}

#label_filter_anwenden:hover{
background:white;
border-radius:3px;
border-style:solid;
border-width:1.5px;
border-color:rgba(137,136,191,250);
color: black;
}

#label_filter_loadimage{
background:#03013A;
border-radius:5px;
border-style:solid;
border-width:1.5px;
border-color:black;
color: white;
}

#label_filter_loadimage:hover{
background:white;
border-radius:5px;
border-style:solid;
border-width:1.5px;
border-color:#03013A;
color: #03013A;
}




#label_change_display{
background:transparent;
border-style:solid;
border-color:black;
color: white;
}

#comboBox_filter{}


#label_img_before{
background:white;
border-style:solid;
border-color:black;
border-width:3px;
}


#label_img_after{
background:white;
border-style:solid;
border-color:black;
border-width:3px;
}

QComboBox{
border-style:solid;
border-color:black;
border-width:0.5px;
border-radius:5px;
}

QComboBox::drop-down{
border-width:0px;
}

QComboBox::down-arrow{
color:green;
}

QRadioButton{
background: rgba(27,27,27,30);
border-style:solid;
border-width:0.5 px;
border-color:grey;

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_line_1">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>70</y>
       <width>649</width>
       <height>2</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.Farah PUA</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_img_before">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>110</y>
       <width>281</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="GUI_Ressources.qrc">:/GUI_Ressources/image_test_before.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_img_after">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>281</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="GUI_Ressources.qrc">:/GUI_Ressources/image_test_after.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_title_before">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Before</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_title_after">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>100</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>After</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QFrame" name="frame_similarity">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>420</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="comboBox_method">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>11</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="maxVisibleItems">
       <number>20</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>rms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>max-norm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>binary</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_result_display">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_choose_filter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Method:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Result:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_aktualisieren">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Aktualisieren</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_choose_filetype">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_filetype">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Image</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Diagram</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_title_similarity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.Farah PUA</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Similarity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_filter">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>420</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_filter_anwenden">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Anwenden</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_title_filter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.Farah PUA</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_blur">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Blur</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_invert">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Invert</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_preventfd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prevent FD</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_logo">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>10</y>
       <width>271</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="GUI_Ressources.qrc">:/GUI_Ressources/Facedetect_logo2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame_facedetection">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_title_facedetection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.Farah PUA</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Face Detection</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_choose_facedetection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Method:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_facedetection">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>51</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>11</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="maxVisibleItems">
       <number>20</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>Face Detection 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Face Detection 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Face Detection 3</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLabel" name="label_filter_loadimage">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>370</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_line_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>385</y>
       <width>649</width>
       <height>2</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.Farah PUA</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
    <zorder>label_logo</zorder>
    <zorder>frame_similarity</zorder>
    <zorder>label_img_before</zorder>
    <zorder>label_img_after</zorder>
    <zorder>label_title_before</zorder>
    <zorder>label_title_after</zorder>
    <zorder>frame_filter</zorder>
    <zorder>label_line_1</zorder>
    <zorder>frame_facedetection</zorder>
    <zorder>label_line_2</zorder>
    <zorder>label_filter_loadimage</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="GUI_Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
